### PR TITLE
Release 1.0.3-4

### PR DESCRIPTION
**Pull request by sdruskat**
This is the final release of CFF in the old repository structure and documentation generation.

### CODE REVIEW

**Comment by sdruskat** [comment]
@jspaaks Please check if you're happy with being listed as the author, and double-check if there's anything you can think of that should be part of the release (not that I think there's much to add). Thanks!

File: 1.0.3/specifications.md [hunk 1]
```diff
@@ -19,8 +20,8 @@ abstract:
   format for [CodeMeta](https://codemeta.github.io/) [JSON](https://github.com/codemeta/codemeta/blob/master/codemeta.json) files.
 geometry: margin=2cm
 version: 1.0.3
-specs-version: 1.0.3-3
-release-date: 29 Aug 2018
+specs-version: 1.0.3-4
+release-date: 22 Oct 2019
```
Comment (jspaaks) [inline 1.0.3/specifications.md]: we could update this date to the date when the button is pressed

**Review by jspaaks — APPROVED** [review]
Just a small comment about the value of the date, see comment below.

I'm glad we are simplifying the release process BTW :)

File: 1.0.3/specifications.md (hunk 1 above)
Comment (sdruskat) [reply 1.0.3/specifications.md]:
```suggestion
release-date: 04 Nov 2019
```

**Pull request merged**